lib/update/cluster/phases/bootstrap.go

```diff
@@ -548,6 +576,49 @@ func (p *updatePhaseBootstrap) addUpdateRuntimePackageLabel() error {
 	return nil
 }
 
+// migrateInfluxDBData copies all from /var/lib/data/influxdb if it exist on the node and if destination
+// directory /var/lib/gravity/monitoring/influxdb empty
+func (p *updatePhaseBootstrap) migrateInfluxDBData(ctx context.Context) error {
+	newDataDirectory := "/var/lib/gravity/monitoring/influxdb"
```

Comment:
I'd consider moving these to lib/defaults. Or at least make these local constants instead of variables.

Also wouldn't hurt to add a comment that "oldDataDirectory" is within planet.